src/index.html

```diff
@@ -1,12 +1,319 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#">
+        <img class="logo"
+          src="images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="is-active nav__link"> Apple </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a href="#"
+              class="nav__link"
+              data-qa="nav-hover">
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="card" data-qa="card">
```

Comment:
Consider will be better if you use the semantic tad for card 
```suggestion
      <article class="card" data-qa="card">
```